sdk/eventhub/event-hubs/CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## 5.0.3 (Unreleased)
 
+- Fixes an issue where known properties were not being set on a received event's `systemProperties`.
```

Comment:
```suggestion
- Fixes [issue #7801](https://github.com/Azure/azure-sdk-for-js/pull/7973) where known properties were not being set on a received event's `systemProperties`.
```

I wonder if we should call this out as a "feature request". It didn't exist in previous versions of the library either

Reply:
Sounds fair enough. I can change the wording and update the version accordingly.

Reply:
Updated